### ISSUE TITLE
Fix katacoda-cli validation error

### ISSUE THREAD

**Issue by AndreasAugustin** [katacoda/scenario-examples#14]
Currently when run [katacoda-cli](https://www.katacoda.com/cli) you get errors:
- Scenario=/app/environment-usages/minikube/ - Validation Error: There are missing scenario files step2.md,step3.md
- Scenario=/app/uilayouts/uilayout-iframe/ - Validation Error: Layout specified in the index.json is not part of the acceptable layouts as specified here: https://katacoda.com/docs/scenarios/layouts
- Scenario=/app/uilayouts/uilayout-terminal-iframe-split/ - Validation Error: Layout specified in the index.json is not part of the acceptable layouts as specified here: https://katacoda.com/docs/scenarios/layouts

Whole output:
```bash
$ katacoda validate:all --repo=/app/

Scenario=/app/clipboard/ - is valid
Scenario=/app/create-course/ - is valid
Scenario=/app/create-scenario-101/ - is valid
Scenario=/app/credit-where-credit-due/ - is valid
Scenario=/app/dashboard-tabs/ - is valid
Scenario=/app/dashboard-tabs-iframe/ - is valid
Scenario=/app/displaying-images/ - is valid
Scenario=/app/displaying-progress-spinner/ - is valid
Scenario=/app/displaying-web-ui/ - is valid
Scenario=/app/environment-usages/docker/ - is valid
Scenario=/app/environment-usages/docker-swarm/ - is valid
Scenario=/app/environment-usages/kubernetes/ - is valid
Scenario=/app/environment-usages/kubernetes-cluster/ - is valid
Scenario=/app/environment-usages/minikube/ - Validation Error: There are missing scenario files step2.md,step3.md
Scenario=/app/environment-usages/ubuntu/ - is valid
Scenario=/app/markdown-extensions/ - is valid
Scenario=/app/opening-multiple-terminals/ - is valid
Scenario=/app/quiz/ - is valid
Scenario=/app/run-commands-automatically/ - is valid
Scenario=/app/uilayouts/uilayout-editor-terminal/ - is valid
Scenario=/app/uilayouts/uilayout-iframe/ - Validation Error: Layout specified in the index.json is not part of the acceptable layouts as specified here: https://katacoda.com/docs/scenarios/layouts
Scenario=/app/uilayouts/uilayout-terminal/ - is valid
Scenario=/app/uilayouts/uilayout-terminal-iframe/ - is valid
Scenario=/app/uilayouts/uilayout-terminal-iframe-split/ - Validation Error: Layout specified in the index.json is not part of the acceptable layouts as specified here: https://katacoda.com/docs/scenarios/layouts
Scenario=/app/uilayouts/uilayout-terminal-terminal/ - is valid
Scenario=/app/upload-assets/ - is valid
Scenario=/app/verified-steps/ - is valid
Scenario=/app/visualise-docker/ - is valid
```

**Comment by AndreasAugustin** [comment]
Fix error is done with #15 
Added another PR as extension to have a developer environment available #16. That one could also be used to enable some `CI` pipelines.

**Comment by BenHall** [comment]
Thanks @AndreasAugustin ! We appreciate the help here. The Minikube is certainly problematic due to the missing files. The iframe layouts should be supported and we'll enable them on the server-side 

/cc @gsoria
